### ISSUE TITLE
schema

### ISSUE THREAD

**Issue by johnpgarry** [ARM-software/data#1]
Unfortunately the schema used is not consistent with mainline linux perf tool.

**Comment by algrant-arm** [comment]
Yes, the schema is more general as there's information we want to communicate about events that isn't used by Linux perf. The idea is that a script could convert from one format to the other.
